src/workerd/jsg/util.c++

```diff
@@ -240,6 +292,7 @@ DecodedException decodeTunneledException(
 kj::StringPtr extractTunneledExceptionDescription(kj::StringPtr message) {
   auto tunneledError = tunneledErrorType(message);
   if (tunneledError.isInternal) {
+    // TODO(soon): Include an internal error ID in message, and also return the id.
```

Comment:
This is another potential source of "internal error" messages, but the places where we produce the external exception and internal log differ enough that it will be a more involved change.

Looking at how this function is used, it looks like it's mostly for either startup exceptions -- which should largely be caught during worker validation -- or for exceptions reported in preview, so it's probably a less-frequent source of "internal error"s than the others, too.